src/index.ts

```diff
@@ -1 +1 @@
-export * from 'lib/index.js'
+export * from './lib/ManufacturerLogo.js'
```

Comment:
```suggestion
export * from './lib/index.js'
```

Reply:
I'd prefer that we avoid barrel files where possible. I think the main entry point to a lib is the only place we should use this pattern.